src/civetweb.c

```diff
@@ -5856,6 +5861,9 @@ static void handle_static_file_request(struct mg_connection *conn,
 	                "Content-Type: %.*s\r\n"
 	                "Content-Length: %" INT64_FMT "\r\n"
 	                "Connection: %s\r\n"
+	               	"Cache-Control: no-cache, no-store, must-revalidate\r\n"
+			"Pragma: no-cache\r\n"
+			"Expires: 0\r\n"
```

Comment:
I am using caching of static files for many years without a problem.
Why should it be disabled? 
Is there any problem: I would try to reproduce it.